<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>457</width>
    <height>391</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Apples2Oranges</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="1">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="tabsClosable">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="trainTab">
       <attribute name="title">
        <string>Training</string>
       </attribute>
       <widget class="QTextEdit" name="netOutput">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>201</width>
          <height>304</height>
         </rect>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="acceptRichText">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>10</y>
          <width>121</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Neural Network Log</string>
        </property>
       </widget>
       <widget class="QPushButton" name="loadModel">
        <property name="geometry">
         <rect>
          <x>325</x>
          <y>25</y>
          <width>101</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Load Model</string>
        </property>
       </widget>
       <widget class="QPushButton" name="saveModel">
        <property name="geometry">
         <rect>
          <x>225</x>
          <y>25</y>
          <width>101</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Save Model</string>
        </property>
       </widget>
       <widget class="Line" name="line">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>85</y>
          <width>191</width>
          <height>16</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QSpinBox" name="epochBox">
        <property name="geometry">
         <rect>
          <x>330</x>
          <y>133</y>
          <width>71</width>
          <height>24</height>
         </rect>
        </property>
        <property name="prefix">
         <string/>
        </property>
        <property name="maximum">
         <number>999999</number>
        </property>
        <property name="value">
         <number>1</number>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>135</y>
          <width>51</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Epochs:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>100</y>
          <width>91</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Learning Rate:</string>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="learningRateBox">
        <property name="geometry">
         <rect>
          <x>330</x>
          <y>99</y>
          <width>71</width>
          <height>24</height>
         </rect>
        </property>
        <property name="decimals">
         <number>4</number>
        </property>
        <property name="maximum">
         <double>1.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.001000000000000</double>
        </property>
        <property name="value">
         <double>0.250000000000000</double>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>170</y>
          <width>71</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Batch Size:</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="batchBox">
        <property name="geometry">
         <rect>
          <x>330</x>
          <y>168</y>
          <width>71</width>
          <height>24</height>
         </rect>
        </property>
        <property name="prefix">
         <string/>
        </property>
        <property name="maximum">
         <number>500</number>
        </property>
        <property name="value">
         <number>200</number>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>205</y>
          <width>81</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Dataset Split:</string>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="datasetBox">
        <property name="geometry">
         <rect>
          <x>330</x>
          <y>203</y>
          <width>71</width>
          <height>24</height>
         </rect>
        </property>
        <property name="decimals">
         <number>4</number>
        </property>
        <property name="maximum">
         <double>0.999900000000000</double>
        </property>
        <property name="singleStep">
         <double>0.010000000000000</double>
        </property>
        <property name="value">
         <double>0.990000000000000</double>
        </property>
       </widget>
       <widget class="Line" name="line_2">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>235</y>
          <width>191</width>
          <height>16</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QPushButton" name="loadTrain">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>250</y>
          <width>151</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Load Training Data</string>
        </property>
       </widget>
       <widget class="QPushButton" name="train">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>280</y>
          <width>151</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Train Model</string>
        </property>
       </widget>
       <widget class="QPushButton" name="visualizeData">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>310</y>
          <width>151</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Show Visualization</string>
        </property>
       </widget>
       <widget class="QPushButton" name="randomizeModel">
        <property name="geometry">
         <rect>
          <x>225</x>
          <y>56</y>
          <width>201</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Randomize Weights / Biases</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="testTab">
       <attribute name="title">
        <string>Testing</string>
       </attribute>
       <widget class="QPushButton" name="predictButton">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>310</y>
          <width>201</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Select Image For Prediction</string>
        </property>
       </widget>
       <widget class="QLabel" name="dropImage">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>411</width>
          <height>291</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
